.travis.yml

```diff
@@ -1,10 +1,7 @@
 language: php
 
 php:
-  # 5.3.0 is not supported by Travis
-  - 5.3.3
-  - 5.3
-  - 5.4
+  - 5.5.9
```

Comment:
Why test on 5.5.9? Because that's now the lowest version we support in the composer.json, so good to assert that it actually works.